### ISSUE TITLE
cleanlab can be used for cleansing semantic segmentation dataset??

### ISSUE THREAD

**Issue by tub-sgg** [cleanlab/cleanlab#38]
I found all the examples are only for classification?Is there any example for segmentaion or tutorial?

**Comment by tub-sgg** [comment]
What I mean is: I only need to cleansed dataset. I need to clean the dataset.I do not care about the framework like pytorch,tensorflow,maxnet .etc. Can Cleanlab be used directly for that purpose

**Comment by cgnorthcutt** [comment]
CleanLab works with any ML framework. All you need is holdout set predicted probabilities and noisy labels. You can get holdout predicted probabilities using a cross validation, for example. You can use any framework to get those predicted probabilities. I show an example of for CIFAR (in my case, I used PyTorch): https://github.com/cgnorthcutt/cleanlab/tree/master/examples/cifar10

Read from the start until Step 4 (where you'll learn about finding the label errors and removing them from your dataset).

**Comment by jwmueller** [comment]
We will provide tutorial on using cleanlab for semantic segmentation in the near future, stay tuned!
